backend/src/graphql/resolver/UserResolver.test.ts

```diff
@@ -152,7 +153,7 @@ describe('UserResolver', () => {
       it('throws an error', async () => {
         await expect(mutate({ mutation: createUser, variables })).resolves.toEqual(
           expect.objectContaining({
-            errors: [new GraphQLError('User already exists.')],
+            errors: [new GraphQLError(ERRORS[ERRORS.ERR_USER_ALREADY_EXISTS])],
```

Comment:
Here and at similar places we can find a solution where we use the enum constant number instead of the enum constant name.

The frontend would have to be refactored a bit then.